packages/trpc/server/routers/viewer/credits/getAllCredits.handler.ts

```diff
@@ -0,0 +1,41 @@
+import { CreditService } from "@calcom/features/ee/billing/credit-service";
+import prisma from "@calcom/prisma";
```

Comment:
prisma should be used in repository and then trpc handlers should use those repository methods

because (1) we don't want the complexity of mocking prisma for unit tests (2) we want repositories to be the single source of truth ultimately